src/fastapi_swagger2/utils.py

```diff
@@ -571,7 +594,7 @@ def get_swagger2(
             for p in properties:
                 if "anyOf" in properties[p].keys():
                     any_of = properties[p].pop("anyOf")
-                    if len(any_of) <= 2:
+                    if len(any_of) <= 2 and {"type": "null"} in any_of:
```

Comment:
@luliangce I am little unsure on this condition change. Can you please share some sample code which would trigger this condition? Thanks